Rakefile

```diff
@@ -13,7 +13,8 @@ task default: :spec
 RSpec::Core::RakeTask.new(:spec)
 
 namespace :pull do
-
+  
```

Comment:
I believe you only want to include changes to your CVE file in your commit, not this file and travis.yml

Reply:
Also, your build in Travis is failing so you will want to look into that. Not sure if this is the reason why